Sources/TuistScaffold/TemplateGenerator.swift

```diff
@@ -107,11 +107,19 @@ public final class TemplateGenerator: TemplateGenerating {
                 } else {
                     renderedContents = fileContents
                 }
+            case let .directory(path):
```

Comment:
I'd test this in `TemplateGeneratorTests`

Reply:
Added and tested :)